src/app/components/client/dialog/Dialog.module.scss

```diff
@@ -2,48 +2,87 @@
 
 .dialog {
   background-color: $color-white;
-  position: relative;
-  width: calc(100% - (2 * $spacing-md));
-  max-width: $content-lg;
-  max-height: calc(100% - (2 * $spacing-md));
-  overflow-y: auto;
+  border-radius: $border-radius-md;
   display: flex;
   flex-direction: column;
-  align-items: center;
-  gap: $spacing-xl;
-  border-radius: $border-radius-md;
-  padding: $layout-sm;
```

Comment:
I'm not sure if it's intentional, but the existing dialogs now have a lot less padding than they used to have. If unintentional, maybe it's safer to just split this up in two components, a `<HorizontalDialog>` and a `<VerticalDialog>`?

This is [now](https://deploy-preview-3347--fx-monitor-storybook.netlify.app/?path=/story/pages-onboarding--onboarding):

![image](https://github.com/mozilla/blurts-server/assets/4251/56bdf1cc-3189-4ce6-bc8b-4a466aea345d)

and this is [on main](https://fx-monitor-storybook.netlify.app/?path=/story/pages-onboarding--onboarding):

![image](https://github.com/mozilla/blurts-server/assets/4251/b5fd87fd-a3e5-41f9-8115-1003753c3078)

Reply:
Thanks for catching this layout side effect. I tweaked the padding in [e1036fd](https://github.com/mozilla/blurts-server/pull/3347/commits/e1036fd6577769551bf4a8314f503ee40ec2b0f2).